app/views/lists/show.html.slim

```diff
@@ -5,10 +5,20 @@ section.lists
   .list
       .list-name = @list.name
       ul.tasks
-        - if @list.tasks.present?
-          - @list.tasks.each do |task|
-            li.task
-              = link_to "Task #{task.id}: #{task.description}", task_path(task)
+        - if @tasks.present?
```

Comment:
I should have put this in a partial since it's used 2x.  I'll do that before merging.